neofetch

```diff
@@ -1603,13 +1610,27 @@ get_song() {
                 "Linux") get_song_dbus "spotify" ;;
 
                 "Mac OS X")
-                    song="$(osascript -e 'tell application "Spotify" to artist of current track as string & " - " & name of current track as string')"
+                    song="$(osascript <<END
```

Comment:
Can we indent these to match up with the lines above?

Reply:
You mean like this?
```bash
        "spotify"*)
            case "$os" in
                "Linux") get_song_dbus "spotify" ;;

                "Mac OS X")
                    song="$(osascript <<END
                if application "Spotify" is running then
                    tell application "Spotify"
                        artist of current track as string & " - " & name of current track as string
                    end tell
                end if
END
)"
                ;;
            esac
        ;;
```

That's 99 chars wide, so still within the limit.

Reply:
Yeah

Reply:
Fixed them in eb062af — looks like long AppleScript lines can be split with `\` in heredocs, so I was able to compress them a bit more.